addon.xml

```diff
@@ -1,8 +1,9 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<addon id="plugin.program.steam.library" name="Steam Library" version="0.6.3" provider-name="aanderse">
+<addon id="plugin.program.steam.library" name="Steam Library" version="0.7.0" provider-name="aanderse">
     <requires>
         <import addon="xbmc.python" version="2.19.0" />
         <import addon="script.module.requests" version="2.18.4" />
+        <import addon="script.module.requests-cache" version="0.4.13" />
```

Comment:
@AmineI I'm confused because I'm getting this error:
```
2020-05-06 09:01:33.273 T:140295601628928  NOTICE: VideoInfoScanner: Finished scan. Scanning for video info took 00:01
2020-05-06 09:01:36.733 T:140296834149056   ERROR: unmatched parentheses in skin.hassetting(furniture.animatedposters
2020-05-06 09:01:36.749 T:140296834149056   ERROR: ExecuteAsync - Not executing non-existing script script.favourites
2020-05-06 09:01:36.823 T:140294158804736   ERROR: EXCEPTION Thrown (PythonToCppException) : -->Python callback/script returned the following error<--
                                             - NOTE: IGNORING THIS CAN LEAD TO MEMORY LEAKS!
                                            Error Type: <type 'exceptions.ImportError'>
                                            Error Contents: No module named requests_cache
                                            Traceback (most recent call last):
                                              File "/home/aaron/.kodi/addons/plugin.program.steam.library/addon.py", line 2, in <module>
                                                from resources import main
                                              File "/home/aaron/.kodi/addons/plugin.program.steam.library/resources/main.py", line 6, in <module>
                                                import arts
                                              File "/home/aaron/.kodi/addons/plugin.program.steam.library/resources/arts.py", line 7, in <module>
                                                import requests_cache
                                            ImportError: No module named requests_cache
                                            -->End of Python script error report<--
2020-05-06 09:01:36.829 T:140295601628928   ERROR: GetDirectory - Error getting plugin://plugin.program.steam.library/installed?reload=
```
Note that I cleaned out `.pyo` files just in case something funky was happening... still had the above error :confused:

Reply:
For Kodi to properly add dependencies, I think you have to update the plugin the usual way - by installing a zip file of the add-on through Kodi. That's also why I bumped the version number.

I'll be sure to check on a clean Kodi instance to confirm that behavior

Reply:
Oh silly me... I knew that, but forgot! Everything works great, merging. Thank you for this awesome addition to the plugin!

Reply:
My pleasure, as usual ! I'll write a small changelog for all the PRs once these are merged/refused so that they can be published as a 7.0 release on the repo, if you're okay with that

Reply:
Perfect.